components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ExportUtils.java

```diff
@@ -766,6 +770,13 @@ public static void addAPIMetaInformationToArchive(String archivePath, APIDTO api
                     String schemaContent = apiProvider.getGraphqlSchema(apiIdentifier);
                     CommonUtil.writeFile(archivePath + ImportExportConstants.GRAPHQL_SCHEMA_DEFINITION_LOCATION,
                             schemaContent);
+                    GraphqlComplexityInfo graphqlComplexityInfo = apiProvider.getComplexityDetails(apiIdentifier);
+                    if (graphqlComplexityInfo.getList().size() != 0) {
```

Comment:
```suggestion
                    if (graphqlComplexityInfo.getList().size() > 0) {
```

Reply:
Shall we change the condition like this?

Reply:
Is it really necessary? Since size cannot be a negative integer, these two are the same.

Reply:
+1 to keep this as it is. 
Just thought to improve the readability and symantics.